swimlane/core/fields/attachment.py

```diff
@@ -36,6 +42,26 @@ def add(self, filename, stream, content_type=None):
         return attachment
 
 
+    def validate_stream(self, stream, key):
+        if not isinstance(stream, io.IOBase):
+            raise ValueError('{} must be a stream value.'.format(key))
+
+        data = self.get_data(stream)
+
+        if not data or not data.strip() :
+            raise ValueError('{} must not be an empty stream value.'.format(key))
+
+    def get_data(self, stream):
```

Comment:
Could we remove this part of the validation? The more that I think about it, the more I think there might be a case where the user may upload an empty file and it be a valid use case.

Reply:
Sure! I'll delete that!